docs/style.css

```
@@ -0,0 +1,324 @@
+  margin-right: auto !important;
+}
+
+mdx-content > h2,
+#content > h2 {
+  margin-top: 3.5rem !important;
+}
+
+card h2,
+card h3,
+card [class*="title"],
+tile h2,
```

Stylelint (17.9.0)

[error] Unknown type selector "tile" (selector-type-no-unknown)

(selector-type-no-unknown)